panorama_connector.py

```diff
@@ -304,11 +304,13 @@ def _make_rest_call(self, data, action_result):
         config = self.get_config()
 
         try:
-            response = requests.post(self._base_url, data=data, verify=config[phantom.APP_JSON_VERIFY], timeout=DEFAULT_TIMEOUT)
+            response = requests.post(
+                self._base_url, data=data, verify=config[phantom.APP_JSON_VERIFY], timeout=DEFAULT_TIMEOUT)
```

Comment:
`_make_rest_call` should also return the response from the API call
This is so we can use the response to update action result's properties.

see reference https://github.com/search?q=org%3Asplunk-soar-connectors+%22self._make_rest_call%22&type=code